lib/galaxy/tool_util/data/bundles/models.py

```diff
@@ -167,11 +167,11 @@ def _xml_to_data_table_output_column_move(move_elem: Element) -> DataTableBundle
         source_value = ""
     else:
         source_base = source_elem.get("base", None)
-        source_value = source_elem.text
+        source_value = source_elem.text or ""
     target_elem = move_elem.find("target")
     if target_elem is None:
         target_base = None
-        target_value = ""
+        target_value: Optional[str] = ""
```

Comment:
That implies target_values may be None, is that so ?
```
In [1]: from lxml import etree

In [2]: etree.fromstring("<a></a>").text is None
Out[2]: True
```

Reply:
Yes, that seems to be allowed also in the definition of `DataTableBundleProcessorDataTableOutputColumnMove` on line 34.